.github/files/e2e-tests/create-e2e-projects-matrix.sh

```diff
@@ -2,7 +2,7 @@
 
 set -eo pipefail
 
-PROJECTS=('{"project":"Jetpack connection","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/connection","--retries=2"]}' '{"project":"Jetpack pre-connection","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/pre-connection","--retries=2"]}' '{"project":"Jetpack post-connection","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/post-connection","--retries=2"]}' '{"project":"Jetpack sync","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/sync","--retries=2"]}' '{"project":"Jetpack blocks","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/blocks","--retries=2"]}' '{"project":"Boost","path":"projects/plugins/boost/tests/e2e","testArgs":[]}' '{"project":"Search","path":"projects/plugins/search/tests/e2e","testArgs":[]}' '{"project":"VideoPress","path":"projects/plugins/videopress/tests/e2e","testArgs":[]}')
+PROJECTS=('{"project":"Jetpack connection","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/connection","--retries=2"]}' '{"project":"Jetpack pre-connection","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/pre-connection","--retries=2"]}' '{"project":"Jetpack post-connection","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/post-connection","--retries=2"]}' '{"project":"Jetpack sync","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/sync","--retries=2"]}' '{"project":"Jetpack blocks","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/blocks","--retries=2"]}' '{"project":"Boost","path":"projects/plugins/boost/tests/e2e","testArgs":[]}' '{"project":"Search","path":"projects/plugins/search/tests/e2e","testArgs":[]}' '{"project":"VideoPress","path":"projects/plugins/videopress/tests/e2e","testArgs":[]}' '{"project":"Social","path":"projects/plugins/social/tests/e2e","testArgs":[]}')
```

Comment:
Thanks again for working on this @adimoldovan. I just wanted to check that with this change e2e tests for Social will be run when the PR changes the Social plugin?

I was wondering about the code in packages/publicize and js-packages/publicize-components. That would likely cause a version change for Social, but not necessarily, and we would like to run our e2e tests in that situation too.

Reply:
@pablinos Changes to packages/publicize and js-packages/publicize-components should be covered. 
Changes to whatever is defined in `ci.targets` in the tests project `package.json` should trigger the tests for pull requests.

```
		"targets": [
			"plugins/social",
			"packages/publicize",
			"js-packages/publicize-components",
			"tools/e2e-commons"]
```

On push all tests should run (at least this is happening now, we may change that in the future).